### PR TITLE
Product land page for HD Graphics

### PR DESCRIPTION
**Pull request by Debas-31**
## In this milestone of the project:
- Added README file
- Added product land page for HD Graphics

### CODE REVIEW

**Review by wale-prog — COMMENTED** [review]
Hi @Debas-31 ,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Your linters failed

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

68747470733a2f2f6d65646961342e67697068792e636f6d2f6d656469612f444e3149315447684c3833634e78394453522f67697068792e676966


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        <label for="email">Email:</label>
+        <input type="email" name="email"  id="email" placeholder="  user@example.com" required> 
+        <input id="submit" type="submit" value="submit" >
+    </form>
+</div>
+
+</main>
+
+<!-- 
+<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script -->
```
Comment (wale-prog) [inline index.html:52]: Please do not leave commented code in your files as it clusters your code and make it look unprofessional

**Review by wale-prog — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @Debas-31   😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 

Please have a look at my comments below



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Debas-31) [reply index.html:52]: Thank you for the feedback. I will remove the commented code.

**Pull request merged**